packages/ui/babel.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = withTamagui({
```

Comment:
this might need to go into expo/next folder i think, not sure the ui runs babel on build

Reply:
I bet you are right, and just fixed it